### PR TITLE
add openapi templates to repo

### PR DESCRIPTION
**Pull request by wboereboom**
**Description**
add openapi templates to repo

### CODE REVIEW

**Review by zaiddreakh — APPROVED** [review]

File: templates/typescript/licenseInfo.mustache [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ * Adyen NodeJS API Library
+ * Copyright (c) 2022 Adyen B.V.
```
Comment (michaelpaul) [inline templates/typescript/licenseInfo.mustache:15]: I would be nice if we could make this dynamic: 
```suggestion
 * Copyright (c) {{year}} Adyen B.V.
```

Comment (wboereboom) [reply templates/typescript/licenseInfo.mustache:15]: Good suggestion, was not aware this was possible!

File: templates/typescript/licenseInfo.mustache [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
```
[the remark this answers is not in the document]
Comment (wboereboom) [reply templates/typescript/licenseInfo.mustache:13]: I could. It's been policy to add these up to now. What do you think @AlexandrosMor @acampos1916 ?

[the remark this answers is not in the document]
Comment (wboereboom) [reply templates/typescript/licenseInfo.mustache:13]: ok @michaelpaul, I'll remove them 😢

**Review by zaiddreakh — APPROVED** [review]

File: templates/typescript/licenseInfo.mustache (hunk 1 above)
Comment (michaelpaul) [reply templates/typescript/licenseInfo.mustache]: Bummer, maybe we can leave it out then?
```
Copyright (c) Adyen N.V.
```

File: templates/typescript/licenseInfo.mustache (hunk 2 above)
[the remark this answers is not in the document]
Comment (michaelpaul) [reply templates/typescript/licenseInfo.mustache]: Nah, let's keep it simple and leave license information at `LICENSE` file.

**Review by michaelpaul — APPROVED** [review]

**Pull request merged**